ansible_collections/arista/avd/roles/eos_designs/docs/how-to/avd-to-cvaas.md

```diff
@@ -1,8 +1,7 @@
 # The purpose of this tutorial is to review the steps required in order to integrate AVD with Arista CVaaS.
```

Comment:
I think it might be nicer to make this a sentence and not the tile of the page, e.g.:
```
# How to integrate AVD with CVaaS

The purpose of this tutorial is to review the steps required in order to integrate AVD with Arista CloudVision-as-a-Service.```

Reply:
Agreed

Reply:
Agreed